shapash/utils/lime_backend.py

```diff
@@ -35,6 +37,13 @@ def lime_contributions(model,
         np.array or list of np.array
 
     """
+    if is_lime_available is False:
+        raise ValueError(
+            """
```

Comment:
delete "Active Shapley", you can make sentence like "install LIME package to compute"